src/components/Sum.jsx

```diff
@@ -1 +1,7 @@
-// export function Sum() {}
+import React from 'react';
+
+export function Sum({ a = 0, b = 0 }) {
+  return (
+    <p>{`Sum of ${a} and ${b} is ${Number(a) + Number(b)}`}</p>
```

Comment:
Do you need Number(a) and Number(b), you can use just a and b, but i think it's not a mistake

Reply:
Oleksandr is right, those variables will be numbers anyway.
What is more - it's better to make components an arrow function